Simplenote/src/androidTest/java/com/automattic/simplenote/utils/TestDriver.kt

```diff
@@ -0,0 +1,95 @@
+package com.automattic.simplenote.utils
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiObjectNotFoundException
+import androidx.test.uiautomator.UiSelector
+import com.automattic.simplenote.R
+import com.automattic.simplenote.uipages.*
+import org.hamcrest.Matchers
+import org.junit.Assert
+import java.lang.Error
+import java.lang.Exception
+
+object TestDriver {
+
+    //Add var for each page object
+    val allNotesHomePage = AllNotesHomePage()
+    val launchPage = LaunchPage()
+    val loginPage = LoginPage()
+    val signUpPage = SignUpPage()
+
+    // Returns True if Login button is visible
+    fun isUserOnLoginPage(): Boolean {
+
+        if (!loginPage.loginPageLoginButton.exists()) {
+            launchPage.loginButton.click()
+            launchPage.loginEmailButton.click()
+        }
+        return loginPage.loginPageLoginButton.exists()
+    }
+
+    /**
+     * Insert the username and password in the LogIn page and does the login
+     */
+    @Throws(UiObjectNotFoundException::class)
+    fun logInWithCredentials(username: String, password: String): Boolean {
```

Comment:
- Since it's easy to see that credentials are required for the method, having `WithCredentials` included does not look necessary.
- [just a nitpick] In the context of Simplenote, there is no concept of username. Accounts have an associated email and password only.

```suggestion
    fun logIn(email: String, password: String): Boolean {
```

Reply:
sure, I will fix the naming here.